ecchronos-binary/src/bin/ecc-status.py

```diff
@@ -52,6 +52,7 @@ def print_verbose_repair_job(repair_job, max_lines):
     print(verbose_print_format.format("Repaired(%)", repair_job.get_repair_percentage()))
     print(verbose_print_format.format("Repaired at", repair_job.get_last_repaired_at()))
     print(verbose_print_format.format("Next repair", repair_job.get_next_repair()))
+    print(verbose_print_format.format("Reoccurring", repair_job.reoccurring))
```

Comment:
I think "recurring" is the correct term to use rather than reoccurring. AFAICT there is a slight difference between the words and I believe the definition for "recurring" is more correct for the scheduled repairs.